doc/api/worker_threads.md

```diff
@@ -564,6 +564,9 @@ changes:
     occur as described in the [HTML structured clone algorithm][], and an error
     will be thrown if the object cannot be cloned (e.g. because it contains
     `function`s).
+  * `transferList` {Object[]} If one or more `MessagePort`-like objects
+    are passed in `workerData`, a `transferList` is required for those
+    items or [ERR_MISSING_MESSAGE_PORT_IN_TRANSFER_LIST][] will be thrown.
```

Comment:
Maybe reference the `postMessage()` documentation here? This doesn’t just apply to `MessagePort`s, currently it also includes `ArrayBuffer`s and potentially also more objects in the future

Also, we typically wrap referenced error messages in with `` ` ``, see e.g. what we do for `ERR_WORKER_NOT_RUNNING` on this page

Reply:
Like [this](65e1805)?

Reply:
Yeah, I think that’s fine :+1: